### ISSUE TITLE
Posteo, mbsync, mailbox ... contains server's hierarchy delimiter

### ISSUE THREAD

**Issue by fiechr** [LukeSmithxyz/mutt-wizard#658]
I've already found similar issues, but they seemed to be connected to Gmail or the actual error message was different:

I'm using Posteo with named subfolder (not in INBOX, but at the same level as INBOX), Some of the subfolders contain additional subfolders. Somehow this seems to be an issue while syncing the mail:

For every sub-subfolder i get this error message:

`IMAP error: mailbox name Shopping.AliExpress contains server's hierarchy delimiter`

I've read the man page and checked out the config file for isync/mbsync, especially concerning the "Subfolder" and "Flatten" options. But I'm unsure what to change, since it says that if the Flatten option is used, the Subfolder (set to Verbatim by default by mw) option is ignored. But then I'm unsure if neomutt will still accept this...

https://isync.sourceforge.io/mbsync.html

The error message seems odd to me, since the dot actually IS the hierarchy delimiter, so shouldn't it be fine?

Or does Posteo actually don't support "real" IMAP-subfolder and emulates them by using the dot syntax?

Any ideas how to solve this?

Happy to provide more info, if needed.

**Comment by PRESFIL** [comment]
Which version do you use? These issues refer to relatively recent changes.

You can also [try](https://github.com/LukeSmithxyz/mutt-wizard/issues/600#issuecomment-754545455) to check which delimite used by gmail and compare it with `Flatten` setting.

**Comment by fiechr** [comment]
Thanks for your reply!

I'm using the latest -git package from the AUR:
```
$ pacman -Qs mutt-wizard-git
local/mutt-wizard-git 3.2.1.r2.gebb81bb-1
```

This is the answer I get from the server:
`* LIST (\Noselect) "." ""`

**Comment by PRESFIL** [comment]
On the server, the `AliExpress` folder is located **in** the `Shopping` folder or is it one whole name `Shopping.AliExpress`?

If 2) - looks like it an vendor problem. They using `.` as delimiter on server and they shouldn't allow it to be used in the folder name. You can rename folders and tell them if they have an open tracker.

**Comment by PRESFIL** [comment]
And did you create a new config + mailbox after updating `mutt-wizard-git` or is it left over from the previous version?

**Comment by fiechr** [comment]
Yes, I created a fresh config.

The folders (in Thunderbird or other Mail apps) are hierarchical, so there is a `Shopping` folder and **inside** of it, is a `AliExpress` folder.

But I found out something interesting. I've wrote a config file for mbsync manually with my own folders, and this works perfectly fine! There are no errors and the folder structure gets created as expected (folders inside folders).

Since I tried that on another system with Ubuntu, I've installed isync 1.3.0 from the repos (while Arch uses 1.3.5), which seems to be an older -- but for some reasons somehow working -- version:

```
IMAPStore PosteoIMAP
Host posteo.de
Port 993
User xxxxxx.xxxxxx@posteo.de
PassCmd "pass posteo"
AuthMechs LOGIN
SSLType IMAPS
CertificateFile /etc/ssl/certs/ca-certificates.crt

MaildirStore PosteoMaildir 
Path ~/.Mail/Posteo/ 
Inbox ~/.Mail/Posteo/INBOX 
Subfolders Verbatim

Channel Posteo
Master :PosteoIMAP:
Slave :PosteoMaildir:
Create Both
Expunge Both
Patterns *
SyncState *
ExpireUnread no
```

So I guess I need to take this upsteam?
